qtpy/tests/test_qtgui.py

```diff
@@ -61,6 +61,39 @@ def test_enum_access():
     assert QtGui.QIcon.Normal == QtGui.QIcon.Mode.Normal
     assert QtGui.QImage.Format_Invalid == QtGui.QImage.Format.Format_Invalid
 
+
+@pytest.mark.skipif(
+    sys.platform.startswith('linux') and not_using_conda(),
+    reason="Fatal Python error: Aborted on Linux CI when not using conda")
+@pytest.mark.skipif(
+    sys.platform == 'darwin' and sys.version_info[:2] == (3, 7),
+    reason="Stalls on macOS CI with Python 3.7")
+def test_QMouseEvent_pos_functions(qtbot):
+    """Test `QMouseEvent.pos` and related functions obsolete in Qt6,
+       test `QMouseEvent.position` and related functions missing from Qt5"""
+
+    class Window(QtWidgets.QMainWindow):
+        def mouseDoubleClickEvent(self, event: QtGui.QMouseEvent) -> None:
+            assert event.globalPos() - event.pos() == self.mapToParent(QtCore.QPoint(0, 0))
+            assert event.pos().x() == event.x()
+            assert event.pos().y() == event.y()
+            assert event.globalPos().x() == event.globalX()
+            assert event.globalPos().y() == event.globalY()
+            assert event.position().x() == event.pos().x()
+            assert event.position().y() == event.pos().y()
+            assert event.globalPosition().x() == event.globalPos().x()
+            assert event.globalPosition().y() == event.globalPos().y()
+
+            event.accept()
+
+    window = Window()
+    window.show()
+
+    QtCore.QTimer.singleShot(100, lambda: qtbot.mouseMove(window, QtCore.QPoint(42, 6 * 9)))
```

Comment:
Nice

Reply:
Now even nicer, except for that nasty `ERROR module 'importlib_metadata' has no attribute 'distributions'`, which started appearing randomly this week. It's not a `QtPy` error.

It turned out that the functions, which control the mouse, can start as soon as the window appears. And I don't ever need to close it.

Reply:
> Now even nicer, except for that nasty `ERROR module 'importlib_metadata' has no attribute 'distributions'`, which started appearing randomly this week. It's not a `QtPy` error.

Presumably something in your environment depends on `importlib_metadata`, but the installed version is too old (and is presumably not constrained properly by the version of that dep you have installed, assuming your env is consistent). If you're using a conda-based distribution, could also be the result of pip-conda mixing or similar issues.

Reply:
It's not in my environment. It's here, on GitHub: https://github.com/spyder-ide/qtpy/actions/runs/4210802198/jobs/7308717859

Later runs have not reproduced the error, although I messed up with the branch, and I'm sorry about it.

Reply:
Ah, I see, thanks—must have been a transient transitive dep issue.